scripts/contest.sh

```diff
@@ -23,10 +23,18 @@ if [ ! -f ${ROOT}/bundle.tar.gz ]; then
 fi
 touch ${LOGFILE}
 
+# TODO: This is a temporary change to exclude net_devices test for non-youki runtimes
```

Comment:
You can skip tests individually by using the `is_runtime_runc` function within each test.